pkg/formatter/type.go

```diff
@@ -10,12 +10,14 @@ import (
 	"time"
 )
 
+const DefaultOutputFormat = "text"
```

Comment:
should be private, i.e. starts with lower-case

Reply:
ok, I see that you're using this constant in query.go
Why didn't you define constants for all formats? If you decide to improve something - do it all the way.

Note, the test in CLI help contains quotes where it refers to the string values. For example:
```
 -o, --output string                 Output format in which to display the query results -
                                      "text" | "csv" | "json". (default "text")
```
There is no need to use quotes in CLI, so why did you do that?

Reply:
1. I don't think there's a need for consts for all supported types. I think we should, however, have a const to define what's the default value, as we have for other default values, which is what I added. (Also, it's not my job to improve the CLI code and it's OK to improve it in stages ... .)
2. While the quotes around string CLI arguments aren't required (unless the values have spaces), they're not wrong, and I prefer to add them in the CLI help for clarity, especially seeing as there's no text formatting in the CLI. This is also consistent with the Cobra's behavior, as it adds quotes around the default values of string flags (e.g., `(default "text")` in this case was added by Cobra). I did omit the strings from the supported aggregate functions list (after going back and forth) because there the values are names of functions that the final flag string can optionally contain (+ it's a long list and adding the quotes seems to cumbersome).